src/js/main.js

```diff
@@ -1,3 +1,5 @@
 import {} from "./pages/calc.js";
 import {} from "./pages/timer.js";
```

Comment:
для таких импортов используется вот такая запись
```suggestion
import "./pages/calc.js";
import "./pages/timer.js";
```

Reply:
это то же самое